packages/stable/src/api/documents/types.gen.ts

```diff
@@ -325,16 +428,12 @@ export type DocumentFilterValueList = DocumentFilterValue[];
  */
 export interface DocumentGeoJsonGeometry {
   /** Coordinates of the geometry. */
-  coordinates?:
-    | number[]
-    | number[][]
-    | number[][][]
-    | (number[] & number[][] & number[][][]);
+  coordinates?: number[] | number[][] | number[][][];
```

Comment:
wait, is this a breaking change? could a typescript expect confirm?

Reply:
the new definition is correct `number[] | number[][] | number[][][];`
the old one was faulty, it'd never be satisfied essencially

Reply:
<img width="644" alt="Screenshot 2025-02-07 at 09 46 18" src="https://github.com/user-attachments/assets/666fcb52-a23c-43c0-955a-3af92d6061b8" />